homeassistant/components/zwave.py

```diff
@@ -165,6 +170,10 @@ def start_zwave(event):
         """ Called when Home Assistant starts up. """
         NETWORK.start()
 
+        polling_interval = config[DOMAIN].get(CONF_POLLING_INTERVAL, None)
+        if polling_interval is not None:
+            NETWORK.setPollInterval(polling_interval)
```

Comment:
I'm not too familiar with zwave. Is this something we should do on a network level or should the meter sensor call update on the value from time to time?